src/swap/components/SwapProvider.tsx

```diff
@@ -173,6 +177,7 @@ export function SwapProvider({
       handleError({ swapError: undefined });
 
       try {
+        /* fetch the transaction */
```

Comment:
Line comments are done by using `//`
The `/*` it's only used as multi-line comments at the beggining of a function.